CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Enchancement: Support @SentrySpan and @SentryTransaction on classes and interfaces. (#1243)
 * Enchancement: Do not serialize empty collections and maps (#1245)
 * Ref: Simplify RestTemplate instrumentation (#1246)
+* Enchancement: SentryExceptionResolver should not send handled errors by default (#1248).
```

Comment:
what's about introducing a breaking change section in the changelog when we do such behavior changes?

Reply:
Added a separate section.